src/domain/home/form/HomePreliminaryForm.tsx

```diff
@@ -9,133 +9,153 @@ import Button from '../../../common/components/button/Button';
 import InputField from '../../../common/components/form/fields/input/InputField';
 import {
   validateEqual,
-  validateBirthDay,
   validateRequire,
+  validateBirthDay,
 } from '../../../common/components/form/validationUtils';
 import BirthdayFormField from './partial/BirthdayFormField';
 import { setFormValues } from '../../registration/state/RegistrationActions';
 import { RegistrationFormValues } from '../../registration/types/RegistrationTypes';
+import { defaultRegistrationData } from '../../registration/state/RegistrationReducers';
 
 interface HomeFormValues {
-  childBirthdayDay: number | string;
-  childBirthdayMonth: number | string;
-  childBirthdayYear: number | string;
-  childHomeCity: string;
+  child: {
+    birthday: {
+      day: string | number;
+      month: string | number;
+      year: string | number;
+    };
+    homeCity: string;
+  };
   verifyInformation: boolean;
-  childBirthday?: string;
+  childBirthDay?: string;
```

Comment:
Birthday is one word, no need to change.

Reply:
good one!